src/types/types.ts

```diff
@@ -1,42 +1,46 @@
+import { THEMES } from '../constants';
+
 declare global {
-    interface Window {
-        electronService: {
-            addOnLoadListener: (
-                cb: (
-                    event: object,
-                    payload: {
-                        trackers: TrackerListType;
-                    }
-                ) => void
-            ) => void;
-            addNotificationsListener: (
-                cb: (
-                    event: object,
-                    payload: {
-                        text: string;
-                    }
-                ) => void
-            ) => void;
-            addOnResetDataListener: (cb: () => void) => void;
-            removeGlobalListeners: () => void;
-            sendRestoreAppMessage: () => void;
-            sendUpdateTrackersListEvent: (
-                trackersList: TrackerListType
-            ) => void;
-        };
-    }
+  interface Window {
+    electronService:
+      | {
+          addOnLoadListener: (
+            cb: (
+              event: object,
+              payload: {
+                trackers: TrackerListType;
+              }
+            ) => void
+          ) => void;
+          addNotificationsListener: (
+            cb: (
+              event: object,
+              payload: {
+                text: string;
+              }
+            ) => void
+          ) => void;
+          addOnResetDataListener: (cb: () => void) => void;
+          removeGlobalListeners: () => void;
+          sendRestoreAppMessage: () => void;
+          sendUpdateTrackersListEvent: (trackersList: TrackerListType) => void;
+        }
+      | undefined;
+  }
 }
 
 type TrackerObjectType = {
-    id: string;
-    name: string;
-    isActive: boolean;
-    startedAt: number;
-    stoppedOn: string | null;
-    resumedAt?: number;
-    stoppedOnParsed?: string;
+  id: string;
+  name: string;
+  isActive: boolean;
+  startedAt: number;
+  stoppedOn: string | null;
+  resumedAt?: number;
+  stoppedOnParsed?: string;
 };
 
 type TrackerListType = TrackerObjectType[];
 
-export { TrackerObjectType, TrackerListType };
+type Theme = (typeof THEMES)[keyof typeof THEMES];
```

Comment:
For better readability, this line can be broken as follows:
type Theme = typeof THEMES;
type ThemeRecord = Record<keyof Theme, string>;
!This is an example, you will need to change the names of changes in the example, or imports in the project